### ISSUE TITLE
Taskbar theme development status

### ISSUE THREAD

**Issue by rocksdanister** [rocksdanister/lively#1569]
There are many [reported bugs](https://github.com/rocksdanister/lively/labels/taskbar) with taskbar theming.

Taskbar theming, similar to animated wallpaper is not officially supported by Windows and as such it is prone to breaking.

To make matters more difficult upcoming Windows update is [changing the implementation.](https://github.com/TranslucentTB/TranslucentTB/issues/537)

Since it is becoming increasingly difficult to maintain this feature and its not the main focus of Lively, I am unfortunately putting taskbar theme feature development on hold for now (bugfixes and enhancements.)

As always Lively is an opensource project and development support is always more than welcome. If anyone in the community would like to take over the development and maintenance of taskbar theme feature please feel free to reach out here or through my socials.

If the feature remains in hold, it will be discontinued and I will explore other options such as to add support for third party applications like [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) maintained by my friend @sylveon.

**Comment by sidd1908** [comment]
thank you for the information 
will try translucent tb

**Comment by F-1y** [comment]
thank you.
But I can't imagine Lively without taskbar theming
it's amazing.
but we understand you ...


thank yooooou ....

**Comment by LostOfSpace** [comment]
When you open the start menu, the taskbar changes color by a second and returns it again. Also, the "smooth" mode does not look well with the standard start menu, that is, the colors are different, and do not fit. **I also thank the developers for such a wonderful application and wish you the best of luck in its further development:)**

**Comment by exodus3155** [comment]
Thank you for this app, it is amazing. The option for visualization is it available still? 
I cannot seem to find it in the latest app? Link to the video showing it below.  Thanks 
https://www.youtube.com/watch?v=zU8fFkHIZXI

**Comment by Enigma1309** [comment]
please don't remove this feature just add a experimental tag to it , it would be great

**Comment by abusayed0206** [comment]
I just want to make it 'clear' on the desktop but 'blur' when any application is running. I used to do that with [TranslucentTB ](https://github.com/TranslucentTB/TranslucentTB) when Lively taskbar settings were off.  But, when I changed the wallpaper(video wallpaper) on Lively this behavior is gone! Now, I can't make that behavior again with [TranslucentTB ](https://github.com/TranslucentTB/TranslucentTB). Do I need to reboot the PC or need to do something else?

**Comment by TanyaTheEvil** [comment]
I don't understand why you need to disable this feature? It works great for me. All I need is to make the taskbar clear and it works great. I have windows 11 and the taskbar is too big so I use this feature to make it clear. It works great. Just becuase other features don't work doesn't mean you have to eliminate it for people like me that need it. Also is there another app out there that can make the taskbar clear? I googled it and can't find anything

**Comment by TanyaTheEvil** [comment]
Just so everyone knows that  TranslucentTB  does not work at all. Knothing works except Lively Wallpaper. Please take the hold off of the feature that makes the taskbar clear

**Comment by gabrie1lira** [comment]
I only used "lively" because of this feature, now I see it has been disabled and I see no reason to use it anymore. If you come back, please let me know.

**Comment by camezcua12** [comment]
> I don't understand why you need to disable this feature? It works great for me. All I need is to make the taskbar clear and it works great. I have windows 11 and the taskbar is too big so I use this feature to make it clear. It works great. Just becuase other features don't work doesn't mean you have to eliminate it for people like me that need it. Also is there another app out there that can make the taskbar clear? I googled it and can't find anything

I don't think he said anything about disable or eliminate the feature. Just he won't be putting his available time for this project on that feature. The feature will likely stay there, just won't receive any improvements or bugfixes in the close future.

**Comment by TanyaTheEvil** [comment]
> > I don't understand why you need to disable this feature? It works great for me. All I need is to make the taskbar clear and it works great. I have windows 11 and the taskbar is too big so I use this feature to make it clear. It works great. Just becuase other features don't work doesn't mean you have to eliminate it for people like me that need it. Also is there another app out there that can make the taskbar clear? I googled it and can't find anything
> 
> I don't think he said anything about disable or eliminate the feature. Just he won't be putting his available time for this project on that feature. The feature will likely stay there, just won't receive any improvements or bugfixes in the close future.

Yeah, but it on hold and to me it is one of the best features of lively wallpaper. I can't stand the large taskbar. Lively wallpaper is the only app the makes the taskbar clear. No other app or nothing will make the taskbar clear in windows 11

**Comment by s0ul-less** [comment]
> 

> There are many [reported bugs](https://github.com/rocksdanister/lively/labels/taskbar) with taskbar theming.
> 
> Taskbar theming, similar to animated wallpaper is not officially supported by Windows and as such it is prone to breaking.
> 
> To make matters more difficult upcoming Windows update is [changing the implementation.](https://github.com/TranslucentTB/TranslucentTB/issues/537)
> 
> Since it is becoming increasingly difficult to maintain this feature and its not the main focus of Lively, I am unfortunately putting taskbar theme feature development on hold for now (bugfixes and enhancements.)
> 
> As always Lively is an opensource project and development support is always more than welcome. If anyone in the community would like to take over the development and maintenance of taskbar theme feature please feel free to reach out here or through my socials.
> 
> If the feature remains in hold, it will be discontinued and I will explore other options such as to add support for third party applications like [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) maintained by my friend @sylveon.

i hope you will find a solution quickly because regular task bar is so boring

**Comment by vansmack** [comment]
> > > I don't understand why you need to disable this feature? It works great for me. All I need is to make the taskbar clear and it works great. I have windows 11 and the taskbar is too big so I use this feature to make it clear. It works great. Just becuase other features don't work doesn't mean you have to eliminate it for people like me that need it. Also is there another app out there that can make the taskbar clear? I googled it and can't find anything
> > 
> > 
> > I don't think he said anything about disable or eliminate the feature. Just he won't be putting his available time for this project on that feature. The feature will likely stay there, just won't receive any improvements or bugfixes in the close future.
> 
> Yeah, but it on hold and to me it is one of the best features of lively wallpaper. I can't stand the large taskbar. Lively wallpaper is the only app the makes the taskbar clear. No other app or nothing will make the taskbar clear in windows 11

There is a registry hack/setting to change the size of the taskbar in Windows 11. I recently did it on an install I did on a PC I built for my son. Just search the internet for change taskbar icon size.

Also, I installed Lively and the clear setting works great for me. However, I am using StartAllBack to get Windows 11 to look more like Windows 7. I even have a Quick Launch bar instead of pinned apps/icons, and made a few registry setting changes like getting back the full menu in File Explorer when you right click on a file. No more having to click more options for me.

**Comment by betroboomin** [comment]
are we supposed to expect the feature to be back?

**Comment by nightmare023** [comment]
i cant clear my taskbar. i put clear on lively wallpaper but its not working

**Comment by Prismaria** [comment]
It's quite a shame that you've felt the need to do this. I still run Windows 10 so the W11 implementation change is irrelevant to me. Like others said, TranslucentTB also doesn't work for me, but this is likely a personal issue in my case.

Luckily, I still have one faithful option to go back to: [ClassicShell](https://classicshell.net/). Would recommend anyone still using W10 to install ClassicShell, which can make your taskbar transparent or blurred. Not sure if it works on W11, but try [this open source fork](https://open-shell.github.io/Open-Shell-Menu/) if it doesn't.

**Comment by LaHn89** [comment]
i can't clear my taskbar

**Comment by Aztr1xe** [comment]
Mine has been like this for the past 2 months now

**Comment by cJ-aY** [comment]
i keep trying to update lively wallpapers 
but it always said there was a problem. Do I need to uninstall some things or is this just a glitch?
@rocksdanister

**Comment by LucidActives** [comment]
I never had any problems with it??  The only thing is when my display changes such as when I would switch from my xbox to pc or whenever I turned my monitor on it would turn purple but a simple fix is just press windows key twice

**Comment by Aztr1xe** [comment]
> I never had any problems with it?? The only thing is when my display changes such as when I would switch from my xbox to pc or whenever I turned my monitor on it would turn purple but a simple fix is just press windows key twice

My PC doesn't let me do anything to the taskbar, no matter what. I've tried factory resetting it and using TB. I'm just starting to think that there is something wrong in general

**Comment by qoull** [comment]
> There are many [reported bugs](https://github.com/rocksdanister/lively/labels/taskbar) with taskbar theming.
> 
> Taskbar theming, similar to animated wallpaper is not officially supported by Windows and as such it is prone to breaking.
> 
> To make matters more difficult upcoming Windows update is [changing the implementation.](https://github.com/TranslucentTB/TranslucentTB/issues/537)
> 
> Since it is becoming increasingly difficult to maintain this feature and its not the main focus of Lively, I am unfortunately putting taskbar theme feature development on hold for now (bugfixes and enhancements.)
> 
> As always Lively is an opensource project and development support is always more than welcome. If anyone in the community would like to take over the development and maintenance of taskbar theme feature please feel free to reach out here or through my socials.
> 
> If the feature remains in hold, it will be discontinued and I will explore other options such as to add support for third party applications like [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) maintained by my friend @sylveon.

ur a bitch

**Comment by xNeonHD** [comment]
> ur a bitch

LMAO shots fired 🤣👍

**Comment by sylveon** [comment]
TranslucentTB now supports the new taskbar - you can combine it with Lively to achieve the same effect now. Both programs shouldn't be causing any issues to each other, though you would need to disable the transparency effect in Lively prior.

**Comment by headphones6** [comment]
thx for the info it was really helpful

**Comment by cJ-aY** [comment]
[like]  Cjay PERKINS reacted to your message:
________________________________

**Comment by Nikitapro0017** [comment]
[like]  Cjay PERKINS reacted to your message:
[…](https://github.com/rocksdanister/lively/issues/1569#)

**Comment by ghost** [comment]
> There are many [reported bugs](https://github.com/rocksdanister/lively/labels/taskbar) with taskbar theming.
> 
> Taskbar theming, similar to animated wallpaper is not officially supported by Windows and as such it is prone to breaking.
> 
> To make matters more difficult upcoming Windows update is [changing the implementation.](https://github.com/TranslucentTB/TranslucentTB/issues/537)
> 
> Since it is becoming increasingly difficult to maintain this feature and its not the main focus of Lively, I am unfortunately putting taskbar theme feature development on hold for now (bugfixes and enhancements.)
> 
> As always Lively is an opensource project and development support is always more than welcome. If anyone in the community would like to take over the development and maintenance of taskbar theme feature please feel free to reach out here or through my socials.
> 
> If the feature remains in hold, it will be discontinued and I will explore other options such as to add support for third party applications like [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) maintained by my friend @sylveon.



> There are many [reported bugs](https://github.com/rocksdanister/lively/labels/taskbar) with taskbar theming.
> 
> Taskbar theming, similar to animated wallpaper is not officially supported by Windows and as such it is prone to breaking.
> 
> To make matters more difficult upcoming Windows update is [changing the implementation.](https://github.com/TranslucentTB/TranslucentTB/issues/537)
> 
> Since it is becoming increasingly difficult to maintain this feature and its not the main focus of Lively, I am unfortunately putting taskbar theme feature development on hold for now (bugfixes and enhancements.)
> 
> As always Lively is an opensource project and development support is always more than welcome. If anyone in the community would like to take over the development and maintenance of taskbar theme feature please feel free to reach out here or through my socials.
> 
> If the feature remains in hold, it will be discontinued and I will explore other options such as to add support for third party applications like [TranslucentTB](https://github.com/TranslucentTB/TranslucentTB) maintained by my friend @sylveon.

i just love it those animated wallpaper. nice and simple. #open source software.

**Comment by fhhhhhfg** [comment]
fsfs

**Comment by JorgelucasBr** [comment]
TranslucentTB esta funcionando junto com o Lively na data de hoje, uso Windows 11.

**Comment by zain1806481** [comment]
You would think windows would get a hint. and give us transparency  and  live wallpapers As part of the windows experience. I think they're mostly focused on AI. I'm tired of using third party apps. Even though lively wallpapers. is one of the best apps I've seen. and I hope this developer keeps challenging himself. Do me a favor, bro, don't quit. The taskbar in the themes and everything. Keep trying.

**Comment by Professor1618** [comment]
But bro we have to use taskbar theme. How can we use? Pls bro.

**Comment by Deng-Xian-Sheng** [comment]
> You would think windows would get a hint. and give us transparency and live wallpapers As part of the windows experience. I think they're mostly focused on AI. I'm tired of using third party apps. Even though lively wallpapers. is one of the best apps I've seen. and I hope this developer keeps challenging himself. Do me a favor, bro, don't quit. The taskbar in the themes and everything. Keep trying.

You're right, but it's also very empty. Please give the author some help, such as money

**Comment by mosesarulvin** [comment]
The Task bar thing is really cool

**Comment by BayekGo** [comment]
i need a fix to the taskbar theme pleaseee

**Comment by Fxxndxn** [comment]
> thank you for the information will try translucent tb

don't try. translucent tb didn't work in my computer(10.0.22631.2338)

**Comment by dzramil** [comment]
> Сообщается о многих [ошибках](https://github.com/rocksdanister/lively/labels/taskbar) с темами панели задач.
> 
> Тема панели задач, похожая на анимированные обои, официально не поддерживается Windows и поэтому может сломаться.
> 
> Что еще больше усложняет ситуацию, предстоящее обновление Windows [меняет реализацию.](https://github.com/TranslucentTB/TranslucentTB/issues/537)
> 
> Поскольку поддерживать эту функцию становится все труднее, и она не является основной задачей Lively, я, к сожалению, на данный момент приостанавливаю разработку функции темы панели задач (исправления ошибок и улучшения).
> 
> Как всегда, Lively — это проект с открытым исходным кодом, и поддержка разработчиков всегда приветствуется. Если кто-то из участников сообщества желает взять на себя разработку и поддержку функции темы панели задач, пожалуйста, свяжитесь с нами здесь или через мои социальные сети.
> 
> Если эта функция останется в силе, она будет прекращена, и я рассмотрю другие варианты, например, добавление поддержки сторонних приложений, таких как [TranslucentTB,](https://github.com/TranslucentTB/TranslucentTB) поддерживаемых моим другом.@sylveon.

спасибо вам за ваш продукт я не знаю как вы зарабатывает если он бесплатный  вам благодарность и уважения  от всего сердца

**Comment by Chi-rife** [comment]
hi guys

**Comment by Chi-rife** [comment]
i just want to ask you something

**Comment by Chi-rife** [comment]
how to creat a compt steam ,.

**Comment by furjac** [comment]
will it be fixed

**Comment by Dilesh23** [comment]
Plz work on it and bring it back , without clear taskbar there is no fell for a live wallpaper . Jai Shree RAM

**Comment by nogames5613** [comment]
alternatively, you can just hide your taskbar and it works perfectly

**Comment by iuserain** [comment]
yeah u must close lively then it works

**Comment by Andrew-devcoder** [comment]
I just wanted to make the taskbar transparent...

**Comment by Andrew-devcoder** [comment]
I just wanted to make the taskbar transparent...

**Comment by iuserain** [comment]
Yeah idk but u must download TransparentTB and close Lively.

Andrew Kovpak ***@***.***> schrieb am Mi., 15. Mai 2024,
14:29:

**Comment by AdamKatona** [comment]
Hey, could it be made perhaps optional to be turned on?
In the previous version the main selector with the dropdown worked well at least, now it's buggy too with v2.1.0.8, but I'm willing to live with it bc it's awesome, if you put big letters there, that this feature is not maintained, if there are bugs, they won't be fixed, use it until it works, etc. it wouldn't be a big problem if it's still there is it?

**Comment by rocksdanister** [comment]
New version is stopping the service if Windows build is 10.0.22621.1343 or higher; I have to do this since taskbar architecture changes in that version onwards.

> Hey, could it be made perhaps optional to be turned on?

A flag to override this behavior can be programmed I suppose.

**Comment by shardulkulk297** [comment]
Why we cannot have the option to Selectively cutomize the UI how we want?

**Comment by forzayt** [comment]
I just wanted to make the taskbar transparent...

**Comment by iuserain** [comment]
I don't know!

Shardul Satish Kulkarni ***@***.***> schrieb am So., 7. Juli
2024, 22:15:

**Comment by Adamsh1** [comment]
i don't know to change my Taskbar theme

**Comment by iuserain** [comment]
You can't.

Adamsh1 ***@***.***> schrieb am Do., 12. Sept. 2024, 16:32:

**Comment by BurkenDev** [comment]
Well just take some time and do it, can't be that hard. Maybe do some themeing automation with Lively on Open Shell installs for example. Or just make your own taskbar. Easier. No need to add other features since it's been ok for a while now. So focusing on the taskbar is like the next step.

**Comment by Reddirector** [comment]
sir its been 2 years still on hold!!?

**Comment by rocksdanister** [comment]
> New version is stopping the service if Windows build is 10.0.22621.1343 or higher; I have to do this since taskbar architecture changes in that version onwards.
> 
> > Hey, could it be made perhaps optional to be turned on?
> 
> A flag to override this behavior can be programmed I suppose.

Basically, no one reached out and its impossible for me to maintain this feature due to personal life etc..
